includes/Heading/Heading.php

```
@@ -1,359 +1,268 @@
+		int $columns,
+		int $rows,
+		string $rowColFormat
+	): string {
+		$nstart = 0;
+		$count = count( $articles );
+		$iGroup = $columns !== 1 ? $columns : $rows;
+		$nsize = (int)ceil( $count / $iGroup );
+
+		$output = "{|$rowColFormat\n|\n";
+
+		foreach ( range( 0, $iGroup - 1 ) as $_ ) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$_'. (Unused Code Rules)

(UnusedLocalVariable)